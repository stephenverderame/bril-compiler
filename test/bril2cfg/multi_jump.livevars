digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		main_5 [label="[]\n----------\nx := 42\n----------\n[x]", shape="rectangle", style="rounded"];
		main_6 [label="[x]\n----------\nprint x\n----------\n[]", shape="rectangle", style="rounded"];

		main_0 -> main_5;
		main_5 -> main_6;
	}
}
