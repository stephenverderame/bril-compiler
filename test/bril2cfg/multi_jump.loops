digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		main_5 [label="x := 42", shape="rectangle", style="rounded"];
		main_6 [label="print x", shape="rectangle", style="rounded"];

		main_0 -> main_5;
		main_5 -> main_6;
	}
}
