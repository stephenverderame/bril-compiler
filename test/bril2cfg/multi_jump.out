digraph {
	subgraph cluster_main {
		label="main";
		rankdir="TB";
		0 [label="START", shape="ellipse"];
		5 [label="x := 42", shape="ellipse"];
		7 [label="print x", shape="ellipse"];

		0 -> 5;
		5 -> 7;
	}
}
